image_width: 640
image_height: 480
camera_matrix:
  rows: 3
  cols: 3
  data: [597.600359, 0.000000, 313.451597,
        0.000000, 591.603487, 243.955106,
  	    0.000000, 0.000000, 1.000000]
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.169076, -0.502637, -0.000860, 0.001215, 0.0000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0,  0.0,
         0.0,  1.0, 0.0,
        0.0,  0.0,  1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [597.600359, 0.000000, 313.451597, 0.000000,
       	 0.000000, 591.603487, 243.955106, 0.000000,
  	 0.000000, 0.000000, 1.000000, 0.000000]
